sdk/communication/azure-communication-chat/src/samples/java/com/azure/communication/chat/ReadmeSamples.java

```diff
@@ -45,9 +45,9 @@ public ChatClient createChatClient() {
         NettyAsyncHttpClientBuilder httpClientBuilder = new NettyAsyncHttpClientBuilder();
         HttpClient httpClient = httpClientBuilder.build();
 
-        // Your user access token retrieved from your trusted service
-        String token = "SECRET";
-        CommunicationUserCredential credential = new CommunicationUserCredential(token);
+        // Your user access key retrieved from your trusted service
+        String accessKey = "SECRET";
+        CommunicationUserCredential credential = new CommunicationUserCredential(accessKey);
```

Comment:
Same here, Chat takes a token, not the access key.

(Look out for CommunicationUserCredential which always wraps a token)